packages/rich-text-editor/test/basic.common.js

```diff
@@ -454,6 +468,19 @@ describe('rich text editor', () => {
         });
       });
     });
+
+    it('should parse htmlValue correctly when element is attached but not rendered', async () => {
```

Comment:
Maybe we should create a new test file as there are other tests related that could be moved there too:

https://github.com/vaadin/web-components/blob/f5b1dc8dedaf37007e4fc611ba64a4bf3ba997bf/packages/rich-text-editor/test/basic.common.js#L577

https://github.com/vaadin/web-components/blob/f5b1dc8dedaf37007e4fc611ba64a4bf3ba997bf/packages/rich-text-editor/test/basic.common.js#L600

Reply:
Done.